inductiva/logs/task_logs.py

```diff
@@ -0,0 +1,32 @@
+import websocket
+
+
+def stream_task_logs(task_id):
+    """Stream the logs of a task through a websocket.
+    
+    Args:
+        task_id (int): ID of the task to print the logs of."""
+
+    logs_url = "ws://127.0.0.1:5000/logs"
```

Comment:
Already changed this to an environment variable, since we may need to change this to different urls depending on the stream from production and development.

With that in mind, should all these environment variables move there?